data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -160,7 +160,7 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some dwarves are experts in the sword, mace, and bow and can aid your veterans in teaching others."
```

Comment:
I don't like either of them, but would prefer the old one of the two.

Reply:
Can you make an attempt to switch this text to dwarvish speak?